src/hooks/useTimerStateMachine.ts

```
@@ -0,0 +1,279 @@
+      if (state.status !== 'idle') return state;
+      return {
+        ...state,
+        status: 'running',
+        startTime: action.timestamp,
+        pausedAt: null,
+        lastTick: action.timestamp,
+      };
+
+    case 'PAUSE':
+      if (state.status !== 'running') return state;
+      const elapsedMs = action.timestamp - (state.startTime || action.timestamp);
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)